Casks/wireshark.rb

```diff
@@ -18,15 +18,20 @@
   uninstall_preflight do
     set_ownership '/Library/Application Support/Wireshark'
 
-    if File.read('/etc/group').match?(%r{^access_bpf})
+    if system_command '/usr/bin/dscl', args: ['.', '-read', '/Groups/access_bpf'], must_succeed: false
       system_command '/usr/sbin/dseditgroup',
-                     args: ['-o', 'delete', 'access_bpf'],
+                     args: ['-q', '-o', 'delete', 'access_bpf'],
                      sudo: true
     end
   end
 
   uninstall pkgutil:   'org.wireshark.*',
             launchctl: 'org.wireshark.ChmodBPF'
```

Comment:
Please list all pkgs explicitly because wildcards are not supported here:
```suggestion
  uninstall pkgutil:   [
                         'org.wireshark.ChmodBPF.pkg',
                         'org.wireshark.path_helper.pkg',
                       ],
            launchctl: 'org.wireshark.ChmodBPF'
```

Reply:
@vladimyr yes they are, as shown in the build log:
```
==> Uninstalling packages:
org.wireshark.ChmodBPF.pkg
org.wireshark.path_helper.pkg
```

Reply:
You are right, I confused it with `launchctl`.
@lulor Ignore this one.